android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java

```diff
@@ -61,22 +66,24 @@ static DatePickerDialog createDialog(Bundle args, Context activityContext, @Null
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
       switch (display) {
         case CALENDAR:
-          dialog = new RNDismissableDatePickerDialog(activityContext,
-            activityContext.getResources().getIdentifier("CalendarDatePickerDialog", "style", activityContext.getPackageName()),
-            onDateSetListener, year, month, day);
-          break;
         case SPINNER:
-          dialog = new RNDismissableDatePickerDialog(activityContext,
-            activityContext.getResources().getIdentifier("SpinnerDatePickerDialog", "style", activityContext.getPackageName()),
-            onDateSetListener, year, month, day);
-          break;
-        case DEFAULT:
-          dialog = new RNDismissableDatePickerDialog(activityContext, onDateSetListener, year, month, day);
-          break;
+          String resourceName = display == RNDatePickerDisplay.CALENDAR
+                  ? "CalendarDatePickerDialog"
+                  : "SpinnerDatePickerDialog";
+          return new RNDismissableDatePickerDialog(
+                  activityContext,
+                  activityContext.getResources().getIdentifier(
+                          resourceName,
+                          "style",
+                          activityContext.getPackageName()),
+                  onDateSetListener,
+                  year,
+                  month,
+                  day);
       }
+      return new RNDismissableDatePickerDialog(activityContext, onDateSetListener, year, month, day);
```

Comment:
since you chose to use switch, wouldn't it make more sense to put this into the default branch of the switch? I feel like that would be a little more consistent, and there'd be fewer "levels" from which something is `return`ed.

Reply:
Updated with your suggestion. I didn't see it as a problem initially, but for consistency and clarity of code, it is beneficial refactor